package.json

```diff
@@ -1,38 +1,116 @@
 {
-  "name": "@storyblok/react-workspace",
-  "version": "0.0.1",
+  "name": "@storyblok/react",
+  "type": "module",
+  "version": "4.0.0",
   "private": false,
+  "packageManager": "pnpm@9.13.2",
+  "description": "SDK to integrate Storyblok into your project using React.",
+  "author": "Storyblok",
   "license": "MIT",
-  "workspaces": [
-    "./lib",
-    "./playground",
-    "./playground-next",
-    "./playground-next13-rsc",
-    "./playground-next13-live-editing"
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/storyblok/storyblok-react"
+  },
+  "bugs": {
+    "url": "https://github.com/storyblok/storyblok-react/issues"
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    },
+    "./rsc": {
+      "types": "./dist/rsc/index.d.ts",
+      "import": "./dist/rsc.mjs",
+      "require": "./dist/rsc.js"
+    }
+  },
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
   ],
   "scripts": {
-    "dev": "npm run dev --workspace=lib & npm run demo",
-    "demo": "npm run demo --workspace=playground",
-    "demo-next": "npm run build && npm run demo --workspace=playground-next",
-    "demo-next13-rsc": "npm run demo --workspace=playground-next13-rsc",
-    "demo-next13-live-editing": "npm run demo --workspace=playground-next13-live-editing",
-    "build": "npm run build --workspace=lib",
-    "test": "npm run test --workspace=lib",
-    "test:unit": "npm run test:unit --workspace=lib",
-    "test:e2e": "npm run test:e2e --workspace=lib",
+    "dev": "vite build --watch",
+    "build": "vite build && tsc --project tsconfig.json",
+    "test": "pnpm run test:unit && pnpm run test:e2e",
+    "test:unit": "vitest --watch false",
```

Comment:
@edodusi why the --watch false? Normally this command is used in the development of the unit test.

Reply:
The correct command for CI would be `vitest run` https://vitest.dev/guide/cli#vitest-run

You can set it up similar to this

```
"test:unit": "vitest",
"test:unit:ci": "vitest run",
"test:unit:ui": "vitest --ui",
 ```